custom-scripts/conda_cudatoolkit.sh

```diff
@@ -0,0 +1,2 @@
+#! /bin/bash
+conda install -y cudatoolkit
```

Comment:
Do we always expect to install this via a custom script? Shouldn't this "just happen" if we detect a gpu sku?